spec/pingCodeOwnerSpec.js

```diff
@@ -0,0 +1,104 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
```

Comment:
The test should also be moved to the appropriate test file.

Reply:
On it Sir.